src/entity/src/user.js

```diff
@@ -498,7 +499,13 @@ export default class User {
       client: this.client
     });
 
-    return request.execute()
+    let prm = Promise.resolve();
+
+    if (LiveServiceFacade.isInitialized()) {
+      prm = LiveServiceFacade.unregister();
```

Comment:
I thought about it, but I'm not sure.

Feels like hiding logic somehow. And calling a method, which just does nothing seems odd. I feel if you call it, it should either work, or return an error.

But I am flexible on this one :)